tools/bin/standard_test_pr.sh

```diff
@@ -7,4 +7,5 @@ set -e
 assert_root
 
 ./gradlew --no-daemon --scan -x generateProtocolClassFiles \
-  :airbyte-integrations:connectors:source-github-singer:standardSourceTestPython 
+  :airbyte-integrations:connectors:source-github-singer:standardSourceTestPython \
+  :airbyte-integrations:connectors:source-google-adwords:standardSourceTestPython
```

Comment:
not rate limited?

Reply:
it is. going to revert this once i make sure the test is catapable of passing in the ci.